src/SurveyHandler.ts

```diff
@@ -862,12 +867,22 @@ class SurveyHandler {
                 options += '<span data-rating="' + initial + '" class="option-number-item option-'+ endRange +'-scale-' + initial + ' ' + scaleVisibility + '" style="' + optionStyle + '">' + initial + '</span>';
               }
             }
-            else
-            {
+            else {
+              for(let iterator=0; iterator<=2; iterator++) {
```

Comment:
Instead of for do customMetric.optionCategories.forEach.

This will iterate only if item is there.